Monika After Story/game/script-farewells.rpy

```diff
@@ -1868,3 +1868,22 @@ label bye_prompt_hangout:
     $ persistent._mas_greeting_type_timeout = datetime.timedelta(hours=8)
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_HANGOUT
     return "quit"
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_walk",
+            prompt="I'm going for a walk.",
+            unlocked=True,
+            pool=True
+        ),
+        code="BYE"
+    )
+
+label bye_prompt_walk:
+    m 7wua "How fun, [player]!"
```

Comment:
Is this a typo or supposed to be 'how' not 'have'?

Reply:
intended, she means going for a walk is fun. i'm open to changes tho.